pkg/syfs/syfs.go

```diff
@@ -68,3 +69,9 @@ func ConfigDirForUsername(username string) (string, error) {
 
 	return filepath.Join(u.HomeDir, singularityDir), nil
 }
+
+// GetSysConfigFile returns the path the Singularity system
+// configuration file
+func GetSysConfigFile() string {
```

Comment:
Since we have `SINGULARITY_CONF_FILE` now I wonder whether this function is useful at all.

Reply:
@sashayakovtseva I do not know what is `SINGULARITY_CONF_FILE` and could not find a single reference of it. If you are referring to `SINGULARITY_CONFDIR`, this is something totally different: the *user-specific* configuration file (e.g., `~/.singularity`). Here this is the system level configuration file that is set when building Singularity. This is why initially the code was an internal package, after discussing with other members of the team and coming to consensus. Here it does the exact opposite for a reason I do not know. We even have an issue open to finish the discussion about this #3995 but @mem decided to not wait and put aside what was agreed on so far.

Reply:
`SINGULARITY_CONF_FILE ` is defined during make and equals to `SINGULARITY_CONFDIR /singularity.conf`

Reply:
Yes i found it an updated my comment

Reply:
Not sure what to do with this conversation.

I don't have a particular preference for having this function wrap buildcfg.SINGULARITY_CONF_FILE other than that's a mouthful and a little not Go-like (all caps, underscores).

Reply:
I updated the PR to do without this function. Personally I dislike the way it looks...